src/store/Errors.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  ReactNode,
+  useEffect,
+  useState,
+  createContext,
+  useContext,
+} from 'react';
+
+type UErrorsContext = {
+  errorMessage: string;
+  setErrorMessage: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const ErrorContext = createContext<UErrorsContext>({
+  errorMessage: '',
+  setErrorMessage: () => {},
+});
+
+type Props = {
+  children: ReactNode;
+};
+
+export const useErrorNotifications = () => useContext(ErrorContext);
+
+export const ErrorsProvider: React.FC<Props> = ({ children }) => {
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    if (!errorMessage) {
+      return;
+    }
+
+    const timeout = setTimeout(() => setErrorMessage(''), 3000);
```

Comment:
[Optional fix] minor, but can you please convert `3000` to some nicely named constant

Reply:
Ok!